internal/provisioner/objects/service/service.go

```diff
@@ -345,18 +332,14 @@ func currentContourService(ctx context.Context, cli client.Client, contour *mode
 	return current, nil
 }
 
+// currentContourService returns the current Contour Service for the provided contour.
+func currentContourService(ctx context.Context, cli client.Client, contour *model.Contour) (*corev1.Service, error) {
+	return currentService(ctx, cli, contour.Namespace, contour.ContourServiceName())
+}
+
 // currentEnvoyService returns the current Envoy Service for the provided contour.
 func currentEnvoyService(ctx context.Context, cli client.Client, contour *model.Contour) (*corev1.Service, error) {
-	current := &corev1.Service{}
-	key := types.NamespacedName{
-		Namespace: contour.Namespace,
-		Name:      contour.EnvoyServiceName(),
-	}
-	err := cli.Get(ctx, key, current)
-	if err != nil {
-		return nil, err
-	}
-	return current, nil
+	return currentService(ctx, cli, contour.Namespace, contour.EnvoyServiceName())
```

Comment:
let's drop these and just use the `currentService` func with the right name param where they're used, they're not adding much.

Reply:
Sorry, what I meant here was to delete the `currentContourService` and `currentEnvoyService` function definitions, and wherever they're used, just call `currentService(...)` directly, passing in the appropriate namespaced name for the service.

Reply:
done, sorry, I misunderstood it